src/vs/workbench/common/editor/editorGroupModel.ts

```diff
@@ -434,7 +434,7 @@ export class EditorGroupModel extends Disposable implements IEditorGroupModel {
 				this.doPin(existingEditor, existingEditorIndex);
 			}
 
-			// Activate / select it
+			// Activate / select it TODO why do we call this when `makeActive` is false?
```

Comment:
@benibenj is it intentional that opening inactive editors clears the selection?

Reply:
So the selection can change or be kept when opening an editor as not active. This was one of the feedback you provided